GemmaM0_Headband/code.py

```diff
@@ -0,0 +1,33 @@
+# Gemma M0 Onchip Temperature Sensor
+# Project by Kathy Ceceri
+# CircuitPython by Anne Barela
+# Adafruit Industries, 2019
+
+import time
+import board
+import microcontroller
+import neopixel
+import adafruit_dotstar
+
+# Set up NeoPixel strand
+pixels = neopixel.NeoPixel(board.D1,  # NeoPixels on pin D1
+                           4,         # Number of Pixels
+                           brightness=0.2,   # Change from 0.0 to 1.0
+                           auto_write=True)  # Write strand immediately
```

Comment:
`auto_write=True` kwarg not required, the boolean kwarg is `True` by default. 

see: https://circuitpython.readthedocs.io/projects/neopixel/en/latest/api.html#neopixel.NeoPixel

Reply:
changed